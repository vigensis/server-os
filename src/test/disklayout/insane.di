SCSI	disk0	MFG	SSD	100030242816	no	yes
SCSI	disk1	MFG2	DISK2	600127266816	no	no
SCSI	disk2	MFG2	DISK2	600127266816	no	no
SCSI	disk3	MFG	DISK	3000127266816	no	no
SCSI	disk4	MFG	DISK	3000127266816	no	no
SCSI	disk5	MFG	DISK	3000127266816	no	no
SCSI	disk6	MFG	DISK	3000127266816	no	no
SCSI	disk7	MFG	DISK	3000127266816	no	no
SCSI	disk8	MFG	DISK	3000127266816	no	no
SCSI	disk9	MFG	DISK	3000127266816	no	no
SCSI	disk10	MFG	DISK	3000127266816	no	no
SCSI	disk11	MFG	DISK	3000127266816	no	no
SCSI	disk12	MFG	DISK	3000127266816	no	no
SCSI	disk13	MFG	DISK	3000127266816	no	no
SCSI	disk14	MFG	DISK	3000127266816	no	no
SCSI	disk15	MFG	DISK	3000127266816	no	no
SCSI	disk16	MFG	DISK	3000127266816	no	no
USB	key0	USBMFG	USB Key	15737028608	yes	no
