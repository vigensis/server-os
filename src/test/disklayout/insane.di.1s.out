{
	"spares": [
		{
			"name": "disk9",
			"vid": "MFG",
			"pid": "DISK",
			"size": "3000127266816",
			"solid_state": false
		}
	],
	"vdevs": [
		{
			"type": "raidz3",
			"devices": [
				{
					"name": "disk10",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk11",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk12",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk13",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk14",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk15",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk16",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk3",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk4",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk5",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk6",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk7",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				},
				{
					"name": "disk8",
					"vid": "MFG",
					"pid": "DISK",
					"size": "3000127266816",
					"solid_state": false
				}
			]
		}
	],
	"capacity": 30001272668160,
	"logs": [
		{
			"name": "disk0",
			"vid": "MFG",
			"pid": "SSD",
			"size": "100030242816",
			"solid_state": true
		}
	],
	"cache": [
		{
			"name": "disk1",
			"vid": "MFG2",
			"pid": "DISK2",
			"size": "600127266816",
			"solid_state": false
		},
		{
			"name": "disk2",
			"vid": "MFG2",
			"pid": "DISK2",
			"size": "600127266816",
			"solid_state": false
		}
	]
}
